app/models/moment_mood.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: moment_moods
+#
+#  id                :bigint(8)        not null, primary key
+#  moment_id         :bigint
+#  mood_id           :bigint
+#
+
+class MomentMood < ApplicationRecord
+  belongs_to :moment
+  belongs_to :mood
```

Comment:
You'll also want some uniqueness validation to make sure that there is no duplication of records with the same moment and mood ID.